https://bugs.gentoo.org/919380

Exposed by a change in Meson to stop inhibiting Vala warnings, e.g.
```
umockdev-record.p/src/umockdev-pcap.c: At top level:
umockdev-record.p/src/umockdev-pcap.c:101:20: error: ‘umockdev_ioctl_usb_pcap_handler_properties’ defined but not used [-Werror=unused-variable]
  101 | static GParamSpec* umockdev_ioctl_usb_pcap_handler_properties[UMOCKDEV_IOCTL_USB_PCAP_HANDLER_NUM_PROPERTIES];
      |                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

See https://github.com/mesonbuild/meson/commit/5f659af870011e74299d1455a65c2cd5f5ace51f.
--- a/meson.build
+++ b/meson.build
@@ -21,7 +21,7 @@ add_project_arguments(
   '-Werror=format-security',
   '-Werror=format=2',
   #'-Werror=missing-include-dirs',
-  '-Werror=unused-variable',
+  #'-Werror=unused-variable',
   '-Werror=return-type',
   '-Werror=uninitialized',
   # experimental: first check this on all Debian architectures
